baseband/vdif/header.py

```diff
@@ -74,13 +110,13 @@ def __new__(cls, words, edv=None, verify=True, **kwargs):
 
         # Have to use key "-1" instead of "False" since the dict-lookup treats
         # 0 and False as identical.
-        cls = cls._vdif_edv_header_classes.get(edv if edv is not False else -1,
-                                               VDIFBaseHeader)
+        cls = VDIF_HEADER_CLASSES.get(edv if edv is not False else -1, 
+                                                                VDIFBaseHeader)
```

Comment:
pep8 indentation is off (use emacs with the astropy setup and tab will just move it to the right place)

Also extraneous space at the end of the line above (again, use emacs setup and this gets dealt with automatically)